spec/lib/healthcheck/government_data_check_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+RSpec.describe Healthcheck::GovernmentDataCheck do
+  describe "#status" do
+    context "when government data is unavailable" do
+      it "returns critical" do
+        gov_data_check = described_class.new
+        expect(gov_data_check.status).to be(:critical)
+      end
+    end
+
+    context "when government data is set to expire" do
+      it "gives a warning" do
+        populate_default_government_bulk_data
+        gov_data_check = described_class.new
+        travel_to(7.hours.from_now) do
+          expect(gov_data_check.status).to be(:warning)
+        end
+      end
+    end
+
+    context "when everything is fine" do
+      it "returns ok" do
+        populate_default_government_bulk_data
+        gov_data_check = described_class.new
+        expect(gov_data_check.status).to be(:ok)
+      end
+    end
+  end
+
+  describe "#details" do
+    let(:gov_data_check) { described_class.new }
+
+    it "displays appropriate messages" do
+      government_repo = BulkData::GovernmentRepository.new
+      gov_data_check.instance_variable_set(:@government_repo, government_repo)
+      allow(government_repo).to receive(:cache_age).and_return(1.second)
+      expect(gov_data_check.details[:warning]).to eq("Government data not refreshed in 0 hours 1 minutes")
```

Comment:
Should 1 second really output 1 minute?

Reply:
Have now dropped this entirely after face to face conversation.